pkgs/top-level/all-packages.nix

```diff
@@ -13384,19 +13384,7 @@ with pkgs;
 
   xrdp = callPackage ../applications/networking/remote/xrdp { };
 
-  inherit
-    ({
-      freerdp = callPackage ../applications/networking/remote/freerdp {
-        inherit (darwin.apple_sdk.frameworks) AudioToolbox AVFoundation Carbon Cocoa CoreMedia;
-        inherit (gst_all_1) gstreamer gst-plugins-base gst-plugins-good;
-      };
-      freerdp3 = callPackage ../applications/networking/remote/freerdp/3.nix {};
-    })
-    freerdp
-    freerdp3
-    ;
-
-  freerdpUnstable = freerdp;
+  freerdp3 = freerdp;
```

Comment:
this should also go to `aliases.nix`

Reply:
`aliases.nix` complains about uses of a deprecated alias within nixpkgs. I can put this in aliases.nix, but then i need to treewide replace freerdp3 -> freerdp. If you are fine with that, sure

Reply:
if this is something that should be done, i have no problem doing so in here